src/main/java/com/google/sps/data/MentorshipRequest.java

```diff
@@ -0,0 +1,7 @@
+package com.google.sps.data;
+
+public abstract class MentorshipRequest {
```

Comment:
Do all the classes and interfaces need to be public? Should some have default access level ?